developers/contributor-guide/current/index.md

```diff
@@ -3,29 +3,51 @@ layout: layout-documentation
 
-{% include docs-support-links.html %}
+## Contributing to Weaviate
+
+🎉 First off, thank you for taking the time to contribute! 🎉
+
+We are delighted to have you here. We are thrilled that you want to contribute to the Weaviate core, as together we can make Weaviate even better. We strive to build an engaging community and we encourage you to participate, share your ideas and make friends. As you become more acquainted with the community, there are several initiatives to consider, which are listed below. Everyone is welcome to join our [Slack](https://weaviate.slack.com/) channel and interact with other members of the community.
+
+Please change your Slack workspace display name to your name so that you can connect with other members more easily. 
+
+**Recommend resources**
+
+Here are some resources that will help you familiarize with Weaviate and it's applications:
+
+* [Weaviate Blogs](/blog.html){:target="_blank"} – a series of blog around Weaviate and the overall vector search space.
+* [Weaviate Newsletter](http://weaviate-newsletter.semi.technology/) – bi-weekly newsletter with updates on the latest and greatest announcements.
+* [Awesome Weaviate](https://github.com/semi-technologies/awesome-weaviate){:target="_blank"} – a list of curated examples and tutorials of how to use the Weaviate.
+* [Weaviate Examples](https://github.com/semi-technologies/weaviate-examples){:target="_blank"} – a repository of various examples projects created by the community. Each example shows a different Weaviate use case. You can add your own examples too!
+* [Weaviate's YouTube Channel](https://www.youtube.com/c/SeMI-and-Weaviate/featured){:target="_blank"} – this channel is about all Weaviate and vector search in general. You will find informative podcasts and live demonstration.
+
+# Join Weaviate Community
+
+A lot of community communication happend at the [Weaviate Community Slack](https://join.slack.com/t/weaviate/shared_invite/zt-goaoifjr-o8FuVz9b1HLzhlUfyfddhw){:target="_blank"}. Everyone is welcome to [join](https://join.slack.com/t/weaviate/shared_invite/zt-goaoifjr-o8FuVz9b1HLzhlUfyfddhw){:target="_blank"}.
+
+To get involved in our community, please make sure you to familiarise yourself with the project's [Code of Conduct](https://www.semi.technology/playbooks/misc/code-of-conduct.html).
+
+We understand that navigating a new open source project can be difficult, and that it takes time to become acquainted with the code base. If at any point of time, you face any difficulties, don't hesitate to ask in the `#general` channel on our Slack. Our team and the community will help you solve your problem.
+
+Reach out to members of the community, introduce yourself, and share your ideas/doubts. Tell them about your areas of interest and what technologies you are using to build your projects. The more they know about you, the better they will be able to match project requirements to your interests and abilities.
```

Comment:
Whenever referring to the Weaviate community, I think we should `we/us` rather than `they/them`